init.lua

```diff
@@ -14,6 +33,26 @@ function craftguide:get_recipe(item)
 	return item
 end
 
+function craftguide.extract_groupnames(groupname)
+	if string.sub(groupname, 1, 6) == "group:" then
```

Comment:
What, whaaaat? Where did you hear THAT? You are basically saying that using any core function in the `string` table the “normal” way is “bad practice”. o_O